build.sbt

```diff
@@ -0,0 +1,73 @@
+import sbtrelease.ReleaseStateTransformations._
+
+name:= "content-api-firehose-client"
+organization := "com.gu"
+scalaVersion := "2.11.8"
+scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
+scalacOptions in doc in Compile := Nil
+
+pomExtra := (
+<url>https://github.com/guardian/content-api-firehose-client</url>
+  <developers>
+    <developer>
+      <id>LATaylor-guardian</id>
+      <name>Luke Taylor</name>
+      <url>https://github.com/LATaylor-guardian</url>
+    </developer>
+  </developers>
+)
+publishTo <<= version { v =>
+  val nexus = "https://oss.sonatype.org/"
+  if (v.trim.endsWith("SNAPSHOT"))
+    Some("snapshots" at nexus + "content/repositories/snapshots")
+  else
+    Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+}
+publishMavenStyle := true
+publishArtifact in Test := false
+pomIncludeRepository := { _ => false }
+releasePublishArtifactsAction := PgpKeys.publishSigned.value
+organization := "com.gu"
+licenses := Seq("Apache v2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html"))
+scmInfo := Some(ScmInfo(
+  url("https://github.com/guardian/content-api-firehose-client"),
+  "scm:git:git@github.com:guardian/content-api-firehose-client.git"
+))
+
+releaseProcess := Seq(
+  checkSnapshotDependencies,
+  inquireVersions,
+  runClean,
+  runTest,
+  setReleaseVersion,
+  commitReleaseVersion,
+  tagRelease,
+  publishArtifacts,
+  setNextVersion,
+  commitNextVersion,
+  releaseStepCommand("sonatypeReleaseAll"),
+  pushChanges
+)
+
+scroogeThriftDependencies in Compile := Seq("content-api-models", "story-packages-model-thrift", "content-atom-model-thrift")
+resolvers += "Guardian GitHub Repository" at "http://guardian.github.io/maven/repo-releases"
+
+libraryDependencies ++= Seq(
+  "com.gu" % "content-api-models" % "9.3",
+  "com.amazonaws" % "amazon-kinesis-client" % "1.6.3",
+  "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.11.11",
```

Comment:
Kinesis Client library 1.6.3 does not work with AWS Java SDK 1.11.x

https://github.com/awslabs/amazon-kinesis-client/issues/74